# Exercício de Casa 🏠 

## Teste de Hipótese

Coletar a altura média de cada uma das alunas da on35

**Exercício: Teste z para Altura Média da Turma**

**Cenário:**

Queremos avaliar se a altura média da turma on35 é significativamente diferente da altura média nacional da população feminina brasileira, que é de 161 cm.
 

**Objetivo:**

    Verificar se a altura média da turma é significativamente diferente de 161 cm.

**Dados da Amostra:**

    Altura da turma (em cm): [165,157,163,170,167,161,160,170,157,160,157,168,162,170,167,165,166,158,174,169,163,174,162,170,155,154,158,160,166]

**Perguntas:**


Calcule a estatística *z* e o valor *p* para verificar se a altura média da turma é significativamente diferente de 161 cm.

**Interprete os resultados:** Preciso saber se rejeitou ou não a hipótese nula.
---

## **Formulando as hipósteses:**

A hipótese nula(H₀): A altura média das alunas da turma on35 é igual a média da população feminina nacional, que é 161cm.

A hipótese alternativa (H₁): A média da altura das alunas da turma on35 não é igual a média da população feminina nacional, que é 161cm.



In [7]:
import numpy as np
from statsmodels.stats.weightstats import ztest

#Nível de significância
ns = 0.05


# Dados da amostra
altura_turma = np.array([165,157,163,170,167,161,160,170,157,160,157,168,162,170,167,165,166,158,174,169,163,174,162,170,155,154,158,160,166])
media_nacional = 161
media_alunas = int(np.mean(altura_turma))
# Teste z para uma amostra
z_statistic, p_value = ztest(altura_turma, value=media_nacional)

print(f"Média da altura das alunas: {media_alunas}cm")
print(f"Média da altura da população feminina nacional: {media_nacional}cm")
print(f"z-statistic: {z_statistic:.4f}")
print(f"p-value:{p_value:.4f} ")


if p_value < ns:
    print("Rejeitamos a hipótese nula, não há evidências de que a média da altura das alunas seja igual a média da população feminina nacional")
else:
    print("Não rejeitamos a hipótese nula, há evidências de que a média da altura das alunas é igual a média da população feminina nacional")


Média da altura das alunas: 163cm
Média da altura da população feminina nacional: 161cm
Teste z para uma amostra:
z-statistic: 2.6201
p-value:0.0088 
Rejeitamos a hipótese nula, não há evidências de que a média da altura das alunas seja igual a média da população feminina nacional


***Conclusão:***

Nível de significância usado no testes estatísticos foi 0.05 (ou 5%), e o p-value é de (0.0088) que é menor que 0.05 portanto, rejeitamos a hipótese nula.

***Interpretação Final:***

A altura média da amostra de alturas da turma é significativamente diferente de 161 cm. Em outras palavras, há evidências estatisticamente significativas para afirmar que a altura média da turma difere da altura média da população feminina nacional que é 161 cm.